cmd/argocd/commands/app.go

```diff
@@ -979,6 +980,22 @@ func parseSelectedResources(resources []string) []argoappv1.SyncOperationResourc
 	return selectedResources
 }
 
+func parseLabels(labels []string) map[string]string {
```

Comment:
Minor: but you could write a unit test for this. I don't know if the case when you have `name=foo=bar` needs to be considered.

Reply:
Added a test!